src/move/research/uniswap-v3-whitepaper/sources/structs.move

```diff
@@ -0,0 +1,53 @@
+module research::structs {
+
+    use aptos_framework::table::Table;
+
+    struct Ratio has copy, drop {
+        numerator: u64,
+        denominator: u64,
+    }
+
+    /// An NFT.
```

Comment:
Is this struct only for an NFT? The description in the readme doesn't make it seem like it, but not sure

Reply:
The `Position` struct is an NFT that represents a user's liquidity contribution. In Uni v2 these were fungible tokens, but in v3 they are not fungible. But you can still borrow against them, sell them, etc., hence NFT